app/views/projects/_import_export.html.erb

```diff
@@ -1,13 +1,23 @@
 <details id="import-export">
   <summary class="import-export-header"">
     <i class="fa fa-lg fa-chevron-down rotate"></i>
-    <h4 class="import-export-header__text">CSV Import / Export</h4> </summary>
+    <h4 class="import-export-header__text">CSV Import / Export</h4>
+  </summary>
   <section>
     <div id="import" class="card-deck mb-3">
       <div class="card mb-6 box-shadow">
         <div class="card-header">
           <h4 class="my-0 font-weight-normal">Import CSV</h4>
         </div>
+        <div id="instructions">
+          <span>Instructions:</span>
+          <ul class="list">
+            <li class="list-item">The file must have the following columns and headers: id, title, description and position.</li>
+            <li class="list-item">The column title is required.</li>
+            <li class="list-item">The columns needs to be in the order above.</li>
```

Comment:
Looking at the code, I don't think the order is important since it does `csv["title"]` for example, I doesn't use an index